README.md

```diff
@@ -58,23 +58,23 @@ Future.make(resolve => resolve("hi"))
 /* Shortcut for: let future_A = Future.make(resolve => resolve(99)); */
 let future_A = Future.value(99);
 
-let future_B = future_A |. Future.map(n => n + 1);
+let future_B = future_A->Future.map(n => n + 1);
 
 
 future_A
-|. Future.get(n => Js.log(n)); /* logs: 99 */
+->Future.get(n => Js.log(n)); /* logs: 99 */
 
 future_B
-|. Future.get(n => Js.log(n)); /* logs: 100 */
+->Future.get(n => Js.log(n)); /* logs: 100 */
```

Comment:
Perhaps these examples should be taken from two lines to one?
```reason
future_A->Future.get(n => Js.log(n)); /* logs: 99 */
future_B->Future.get(n => Js.log(n)); /* logs: 100 */
```

Reply:
I agree

Reply:
Adjusted